cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/components/true-cost-chart.js

```diff
@@ -0,0 +1,7 @@
+import trueCostChartImg from "../../img/true-cost-chart.png";
+
+export const TrueCostChart = () => (
+  <div className="m-tool__image-center-half">
+    <img src={trueCostChartImg} />
```

Comment:
This is going to need an `alt` and potentially a `title` attribute for accessibility

Reply:
added alt="True Cost Chart" and title="True Cost Chart"